### PR TITLE
fixed max date range issue

### PR DESCRIPTION
**Pull request by alishobeiri**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "ramda": "^0.23.0",
     "rc-slider": "^6.1.0",
     "react": "^15.4.0",
+    "react-addons-shallow-compare": "^15.6.0",
```
Comment (chriddyp) [inline package.json]: why was this added?

Comment (alishobeiri) [inline package.json]: yeah it's an addon required for `react-dates` package

Comment (alishobeiri) [inline package.json]: if the line isn't there if somebody cloned the repo, they would run into an error when they try to `npm i`

**Comment by chriddyp** [comment]
looks good @alishobeiri - could you update the versions and the changelog? then i'll merge and make a new release. see http://keepachangelog.com/ and http://semver.org/ for more info on changelog format and semver.

**Comment by alishobeiri** [comment]
@chriddyp Should be good now I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.12.2] - 2017-08-17
```
Comment (chriddyp) [inline CHANGELOG.md]: this should be `0.12.3`
